js/web/lib/wasm/jsep/webgpu/ops/transpose.ts

```diff
@@ -13,14 +13,18 @@ export interface TransposeAttributes extends AttributeWithCacheKey {
   readonly perm: number[];
 }
 
-const validateInputs = (inputs: readonly TensorView[]): void => {
+const validateInputs = (inputs: readonly TensorView[], perm: number[]): void => {
```

Comment:
can use `readonly number[]` for `perm` since it's not modified in `validateInputs()`

Reply:
Done